modules/admin_manual/pages/installation/index.adoc

```diff
@@ -1,5 +1,20 @@
 = Installation
 :page-aliases: go/admin-install.adoc
 
-In this section, you will find all the information you need for installing ownCloud. 
+You can install ownCloud in multiple ways, here are our trusted guides:
 
+== xref:installation/manual_installation.adoc[Manual Installation]
```

Comment:
You can not have a reference in parallel being a section. You must have a fixed text for the section and in a new independent line, you can do the reference with xref.

Reply:
You can not or you should not? In my local doc installation it works.